source/lib/storage.js

```diff
@@ -62,6 +62,22 @@ export function getAcknowledgementStatus(): Promise<boolean> {
   return getItemAsBoolean(acknowledgementStatusKey)
 }
 
+const touchIDEnabledKey = 'settings:touchIDEnabled'
```

Comment:
I don't think we need to store the TouchID auth / enabled status in AsyncStorage.